### ISSUE TITLE
formula and style

### ISSUE THREAD

**Issue by Eugene748** [shuchkin/simplexlsxgen#156]
This example creates an error when opening a file in excel. What could be the problem?
```
$data = [
  ['1','100000'],
  ['2','999999'],
  ['3','<style nf="#,##0.00"><f>sum(B1:B2)</f></style>'],
];
SimpleXLSXGen::fromArray($data)->downloadAs('test.xlsx');
```

**Comment by shuchkin** [comment]
try 1.4.12

**Comment by shuchkin** [comment]
use SUM instead sum

**Comment by shuchkin** [comment]
fixed in 1.4.13, when `v=???` skipped default value is not indexed string `formula`, `0` now
